### ISSUE TITLE
requesting an htmx server-side tutorial or example that doesn't use a 'web framework'

### ISSUE THREAD

**Issue by ludicrousdisplay** [bigskysoftware/htmx#3099]
As far as I can tell, all of the Server-Side Examples on the htmx site involve the use of some sort of web-framework. 

Providing a server-side example of the use of htmx with a simple http-server would be well received, given htmx's interest in focus on the simplicity and power of hypertext.

If such an example already exists, then can it be added to the server-side examples page and also placed at the top of the page as a starting point?

Thanks :)

**Comment by ludicrousdisplay** [comment]
https://github.com/AussieGuy0/java-htmx-todo/

**Comment by ludicrousdisplay** [comment]
https://github.com/erwindrsno/simple-to-do-list

**Comment by Telroshan** [comment]
I may be pinpointing, but you say
> all of the Server-Side Examples on the htmx site involve the use of some sort of web-framework

Yet the 2 examples linked above use [Javalin](https://javalin.io/), which describes itself as
> A simple web framework for Java and Kotlin

Not beating the framework allegations! 😄

**Comment by ludicrousdisplay** [comment]
Yes, it is true that Javalin describes itself as a web framework however it is also a 'simple' web framework, and the examples I have found indicate that Javalin can be used as a simple http server with support for mapping handlers to routing.  

I have also looked at sparkjava (which hasn't had any updates since Jetty 9), and spark-11 (which is an updated fork) however these don't seem to have broad support. And I briefly looked at Java's simplehttpserver but to my knowledge it is oracle-licensed so it would be irresponsible of me to suggest that as well. 

If anyone can suggest a simpler approach than Javalin, and that doesn't depend on an Oracle library, please chime in.

**Comment by bennadel** [comment]
This may not be helpful to you; but, I recently took the Contact App from the Hypermedia Systems book and tried to translate it into a ColdFusion context. For the exploration, I'm not using any framework (beyond ColdFusion itself). All of the pages are just simple get/posts back to top-level scripts. https://github.com/bennadel/ColdFusion-HTMX-Contacts-App
